cmd/podmanV2/containers/prune.go

```diff
@@ -0,0 +1,87 @@
+package containers
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/cmd/podmanV2/utils"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	pruneOptions = entities.ContainerPruneOptions{}
```

Comment:
Nit: don't put this in a separate var block.

Reply:
Fixed.